.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,76 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+<!--
```

Comment:
Let's get rid of the HTML comments - this is just confusing for users.

Reply:
I disagree with this one.

Reply:
What do other projects do? Let’s just imitate

Reply:
sklearn uses comments, github recommends markdown

Reply:
Let's stick to GitHub's recommendations. I've only ever seen HTML comments in templates in sklearn.